### ISSUE TITLE
Discussion: Swarm<>YouTube

### ISSUE THREAD

**Issue by ligi** [efdevcon/devcon-website#182]
Currently YouTube is the default and swarm is the second (fallback) option.
Should we turn this around?

**Comment by chrishobcroft** [comment]
Yes

**Comment by wslyvh** [comment]
While I'd be 100% behind switching, the Swarm implementation is currently still somewhat limited as we currently only have 1 encoding. The Etherna team is working on getting multiple version up and better support for different network conditions. Until that time, I think it's better to keep YT as the default

**Comment by chrishobcroft** [comment]
It's perhaps worth highlighting this rn: https://github.com/efdevcon/devcon-website/issues/183 which would resolve the issue with lack of renditions

**Comment by wslyvh** [comment]
Definitely something on our rader. 
As well as other decentralized CDN's (e.g. Fleek net). The Etherna team is also working on something like this as part of the DIP.

Do you have more info about how Livepeer's solution work? Is it decentralized or does it rely on Livepeer Inc's infrastructure?

**Comment by wslyvh** [comment]
Cool. Would you be up to add additional information, implementation details, resources, etc. 
More info on DIP format can be found at https://github.com/efdevcon/DIPs/blob/master/DIPs/DIP-0.md 
I guess that would answer some of the question I posted before as well.

**Comment by ligi** [comment]
I think in this case we do not need a DIP as it is a pure (IMHO uncontentious) software issue.

**Comment by wslyvh** [comment]
Not sure I agree. For this specific issue (making Swarm the default player) you’re right. It should be an easy switch. Although there’s more to it than just flipping a bit, imo.

The Swarm/Etherna option in it’s current state is not sufficient enough in UX, due to available/performance. It only contains an 720p encoding. I know Etherna was working on getting multiple added. I’ll follow up with them for an update. This was all part of their DIP.

The other linked issue from Chris would add a completely new option using Livepeer’ CDN and Arweave. While this sounds interesting, I’d rather see a similar route as Swarm/Etherna and someone leading such a proposal and efforts.

**Comment by ligi** [comment]
> Not sure I agree. For this specific issue (making Swarm the default player) you’re right. It should be an easy switch. Although there’s more to it than just flipping a bit, imo.

yes - that's why I prefixed it discussion - to discuss what needs to be done before switching it around - sure there is work to be done before.

> The Swarm/Etherna option in it’s current state is not sufficient enough in UX, due to available/performance. It only contains an 720p encoding. I know Etherna was working on getting multiple added. I’ll follow up with them for an update. This was all part of their DIP.

let's maybe do it inline and summon @costgallo 

> The other linked issue from Chris would add a completely new option using Livepeer’ CDN and Arweave. While this sounds interesting, I’d rather see a similar route as Swarm/Etherna and someone leading such a proposal and efforts.

sure - I see chris as leading the effort here. Still not sure if for pure software enhancements we need a DIP - but if you want it I will not stay in the way.

**Comment by wslyvh** [comment]
It seems we agree on someone leading these efforts.

The reason I‘m bringing up DIPs, is that it structures the way we discuss/accept these kind of improvements. It makes the process open and fair for others to interact. DIP proposals usually include more details which are useful starting point for discussion.

I do agree that (for software changes) it would be easier to handle it all via issues/PRs. Not switching back/forth between GitHub and forum. In that case, I do feel we should document and ‘formalize’ that process a bit. Some idea/feature request templates could solve this already. Will connect separately on how such a template might look like.

Something else to keep in mind is that if we have IPFS, Swarm and Livepeer/Arweave as a video player option, I don’t think it’s fair anymore to select any as the default player.

**Comment by tmm360** [comment]
Hi, Mirko from Etherna here.
We are definitely working on implementation of multi resolution on Etherna, and we expect to have it available on next months.
We want to provide soon a tool able to scrape more resolutions from youtube (our current source), but for next devcon we want to provide a tool/system able to encode from original source files. Needs to investigate different solutions, probably will be there more than one.
I understand this is a priority for devcon archive, we want to provide all necessary support.
It would be great to switch on next edition Swarm and Youtube, I'm so excited for this! Anyway I'm also happy to be able to support in this way on our first year of live service. Looking for the next year even better! ;)
